src/iexamine.cpp

```diff
@@ -2780,6 +2783,29 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
+{
+    item milk( "milk", 0, 1 );
+
+    // Max duration set here to avoid including effect.h
+    int max_dur = 14400;
```

Comment:
What's that magic number supposed to be? Now I have to look it up and I have to calculate this myself. Why not use `HOURS(24)` or similar?